monkestation/code/datums/status_effects/buffs.dm

```diff
@@ -0,0 +1,7 @@
+/datum/status_effect/hippocratic_oath/on_apply()
+	. = ..()
+	ADD_TRAIT(owner, TRAIT_PERFECT_SURGEON, HIPPOCRATIC_OATH_TRAIT)
```

Comment:
Perfect Surgeon going alongside an override of the Hippocratic Oath for the Rod from miner loot. Nice.

Reply:
Whoops, forgot to put this in the changelog. Did this because I was going to have P.S allow for multiple surgeries too, which is currently bound to the oath effect, so I thought, fuck it, combine it.